package.json

```
@@ -58,6 +58,7 @@
 		"editorconfig-checker": "^5.1.2",
 		"eslint": "^8.56.0",
 		"husky": "^8.0.3",
-		"stylelint": "^16.0.2"
-	}
+		"stylelint": "^16.7.0"
+	},
+	"packageManager": "pnpm@9.5.0+sha512.140036830124618d624a2187b50d04289d5a087f326c9edfc0ccd733d76c4f52c3a313d4fc148794a2a9d81553016004e6742e8cf850670268a7387fc220c903"
```

[ask] Can you tell me how to prevent `packageManager` is automatically inserted by the `pnpm i` command?

---

To do this, you just need to specify the required version without a checksum:
```suggestion
	"packageManager": "pnpm@9.6.0"
```